snowpack/src/util.ts

```diff
@@ -278,24 +278,16 @@ export function findMatchingAliasEntry(
   }
 
   for (const [from, to] of Object.entries(config.alias)) {
-    let foundType: 'package' | 'path' | '' = '';
-    if (isPackageAliasEntry(to)) {
-      if (spec === from || spec.startsWith(`${from}/`)) {
-        foundType = 'package';
-      }
-    } else {
-      if (spec.startsWith(from)) {
```

Comment:
lgtm, I also think this is unnecessary.

Reply:
+1, the same logic now applies to both. I think the reason you weren't able to do this before was that path aliases end with `/` already, but package aliases do not, so this was a good chance to be explicit in this function.

Reply:
my bad，i forgot the `from` has added `/` for relative alias.

Reply:
np! I forgot too, took me a few tries to get this right